### DISCUSSION TITLE
2 Quick Questions About the Installer

### DISCUSSION THREAD

**Discussion by davidi1** [WinMerge/winmerge#1654] [General]
Hello everyone. I have a quick question, during install, a powershell window opens up for a second. It went by so quick I couldn't tell what it did. What exactly did it do? I am on Windows 11 and I do not recall seeing that when I installed this program under Windows 10. Any help would be appreciated!

Also, what is the install option "GnuWin32 Patch for Windows/HTML Tidy/jq" for? What does it do?

**Comment by sdottaka** [comment]
>  I have a quick question, during install, a powershell window opens up for a second. It went by so quick I couldn't tell what it did. What exactly did it do?

Powershell is used to register shell extensions for Windows 11.

> what is the install option "GnuWin32 Patch for Windows/HTML Tidy/jq" for? What does it do?

The above are the commands used by plugins such as ApplyPatch, PrettifyHTML, PrettifyXML, PrettyfyJSON, etc.

**Reply by davidi1** [reply]
Ahh, I see. Thank you :-)
